### PR TITLE
Feature/allow laravel 10

### LINKED ISSUE #429: Laravel 10 Support
- LogViewer Version: 9
- Laravel Version: 10
- PHP Version:8.1

### Description:
can't upgrade to laravel 10

### Steps To Reproduce:
composer update 
### error
 Problem 1
    - Root composer.json requires arcanedev/log-viewer 9.0 -> satisfiable by arcanedev/log-viewer[9.0.0].
    - arcanedev/support 9.0.0 requires illuminate/support ^9.0 -> satisfiable by illuminate/support[v9.0.0, ..., v9.52.0].
    - Only one of these can be installed: illuminate/support[v5.4.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.0, v10.0.0], laravel/framework[v10.0.0]. laravel/framework replaces illuminate/support and thus cannot coexist with it.     
    - arcanedev/log-viewer 9.0.0 requires arcanedev/support ^9.0 -> satisfiable by arcanedev/support[9.0.0].
    - Root composer.json requires laravel/framework ^10.0 -> satisfiable by laravel/framework[v10.0.0].

### PR DESCRIPTION
**Pull request by mihaileu**
fixes #429

### CODE REVIEW

**Review by mudassar1 — APPROVED** [review]
looks good

**Review by dyaskur — APPROVED** [review]

**Review by dyaskur — APPROVED** [review]

**Review by dyaskur — APPROVED** [review]

**Review by dyaskur — APPROVED** [review]
LGTM, hope it will merge  soon

**Review by dyaskur — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -14,16 +14,16 @@
     "type":    "library",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "ext-json": "*",
-        "arcanedev/support": "^9.0",
+        "arcanedev/support": "^9.1",
```
Comment (dyaskur) [inline composer.json:19]: By the way, this is not released yet right?

**Review by dariocarbone — APPROVED** [review]
good, looking forward for an update..

**Comment by arcanedev-maroc** [comment]
Check #433

**Pull request closed without merging**